packages/composites/dns-record-check/src/index.ts

```diff
@@ -2,5 +2,7 @@ import { expose } from '@chainlink/ea-bootstrap'
 import { makeExecute } from './adapter'
 import { NAME } from './config'
 
-const { server } = expose(NAME, makeExecute())
+const adapterContext = { name: NAME }
+
+const { server } = expose(adapterContext, makeExecute())
 export { makeExecute, server }
```

Comment:
Since we are changing this line, could we modify this to be standard with a standard export style?
```suggestion
export { NAME, makeConfig, makeExecute, server }
```